package.json

```diff
@@ -8,6 +8,7 @@
   "scripts": {
     "prepare": "husky",
     "preinstall": "npx only-allow pnpm",
+    "generate-models": "pnpm -r --if-present --stream run generate-models",
```

Comment:
Why are we changing from ':' to '-'?

Reply:
yeah this came from my old fixes/generate-models branch.
It's fixed now